src/index.js

```diff
@@ -375,32 +375,39 @@ class ServerlessIOpipePlugin {
     const debug = createDebugger('createFiles');
     debug('Creating file');
     const { inlineConfig, requireLines } = this.getConfigFromCosmi();
+    const { handlerDir } = this.getOptions();
     const iopipeInclude = `${requireLines}const iopipe = require('${this.getInstalledPackageName()}')(${inlineConfig});`;
     this.funcs.forEach((func, index) => {
       const handler = outputHandlerCode(func);
       const contents = `${iopipeInclude}\n\n${handler}`;
+      fs.ensureDirSync(join(this.originalServicePath, handlerDir));
       fs.writeFileSync(
-        join(this.originalServicePath, `${func.name}-${index}-iopipe.js`),
+        join(
+          this.originalServicePath,
+          `${handlerDir}/${func.name}-${index}-iopipe.js`
```

Comment:
The `/` is OS-specific and will break for Windows users. `path.join()` should handle this for us.

Reply:
Good idea, will edit.